Tests/CartonCLITests/DevCommandTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  File.swift
+//
```

Comment:
**nitpick:** best to remove this bit, as the name doesn't match the actual file name anyway, and we won't have to rename it in the future. Our file headers usually mentions the license instead.

```suggestion
// Copyright 2020 Carton contributors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
BTW, yet another reason I use VSCode is that is supports extensions that insert the license for you automatically, such as this one https://github.com/ymotongpoo/vsc-licenser